fake-api/README.md

```diff
@@ -0,0 +1,75 @@
+# Banking Fake REST API
+Fake JSON API services for banking application
```

Comment:
move this file to `simple-banking-folder

Reply:
done